chassis/invokers/src/main/java/com/griddynamics/jagger/invoker/http/v2/DefaultHttpInvoker.java

```diff
@@ -0,0 +1,28 @@
+package com.griddynamics.jagger.invoker.http.v2;
+
+import com.google.common.base.Preconditions;
+import com.griddynamics.jagger.invoker.InvocationException;
+
+import static java.lang.String.format;
+
+/**
+ * Created by aantonenko on 9/29/16.
+ */
+@SuppressWarnings("unused")
+public class DefaultHttpInvoker extends AbstractHttpInvoker {
+
+    public DefaultHttpInvoker(JHttpClient httpClient) {
+        super(httpClient);
+    }
+
+    @Override
+    public JHttpResponse invoke(JHttpQuery query, JHttpEndpoint endpoint) throws InvocationException {
```

Comment:
But this is a pretty mush default implementation. Without it abstract class looks useless